### PR TITLE
Blog post: announcing two-factor authentication

### PR DESCRIPTION
**Pull request by stsewd**
----
📚 Documentation preview 📚: https://readthedocs-about--333.org.readthedocs.build/

### CODE REVIEW

File: content/posts/two-factor-authentication.md [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+category: Feature announcement
+tags: authentication, security
+authors: Santos Gallegos
+status: published
+image: /images/2fa.jpg
+image_credit: Photo by <a href="https://unsplash.com/@christianw?utm_content=creditCopyText&utm_medium=referral&utm_source=unsplash">Christian Wiediger</a> on <a href="https://unsplash.com/photos/shallow-focus-photography-of-love-lock-6t8PMIfqNVg?utm_content=creditCopyText&utm_medium=referral&utm_source=unsplash">Unsplash</a>
+
+We're excited to announce that two-factor authentication (2FA) is now available for all Read the Docs users!
+
+Two-factor authentication adds an extra layer of security to your account by requiring a second form of verification in addition to your password.
```
Comment (ericholscher) [inline content/posts/two-factor-authentication.md:13]: We should probably note what types of authentication is provided. It's just the 6 digit code currently, right?

**Review by ericholscher — APPROVED** [review]
I think a bit more explaining here would be good, and perhaps a screenshot? :)

**Comment by stsewd** [comment]
> and perhaps a screenshot? :)

Not sure what to put as screenshot, the page asking for the codes? The page showing the user to scan the QR? :thinking:

**Comment by ericholscher** [comment]
I think either works, or we can just not do a screenshot, and just use a stock image for the header image.

**Comment by stsewd** [comment]
> I think either works, or we can just not do a screenshot, and just use a stock image for the header image.

I'm using this image https://github.com/readthedocs/website/blob/9d683296542bd0d676994d74fde1d009e71ecd85/content/images/2fa.jpg, or do you prefer something more 2fa related?

**Comment by humitos** [comment]
> I'm using this image [`9d68329`/content/images/2fa.jpg](https://github.com/readthedocs/website/blob/9d683296542bd0d676994d74fde1d009e71ecd85/content/images/2fa.jpg), or do you prefer something more 2fa related?

That image is _huge_, you should resize it to 1000px wide (similar to others)

**Review by ericholscher — APPROVED** [review]

**Pull request merged**